pkgs/tools/networking/ssl-proxy/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildGo118Module, fetchFromGitHub }:
+buildGo118Module rec {
+  version = "0.2.7";
+  pname = "ssl-proxy";
+
+  src = fetchFromGitHub {
+    owner = "suyashkumar";
+    repo = "ssl-proxy";
+    rev = "v${version}";
+    hash = "sha256-c9BLdDlkrg1z1QrO+vEAVyPtrV/nQcYlGXFmwfAOSpQ=";
+  };
+
+  vendorHash = "sha256-310K9ZSxy/OQ4HYFCcHQaj4NQwzATrOZ2YkhiSkhY5I=";
+
+  testPhase = ''
```

Comment:
??

Reply:
What's the problem?

Reply:
There is no such thing as a `testPhase`. Maybe you meant `checkPhase`?

https://nixos.org/manual/nixpkgs/stable/#sec-stdenv-phases

Reply:
Oh, my bad.